$OpenBSD: patch-_metadata,v 1.1 2011/12/21 20:01:16 jeremy Exp $

Don't have a runtime dependency on rake-compiler, it isn't needed.
Similar change made upstream in hg commit af645318f239.

--- .metadata.orig	Wed Dec 14 11:13:36 2011
+++ .metadata	Wed Dec 14 11:14:18 2011
@@ -59,7 +59,7 @@ dependencies:
     - - ~>
       - !ruby/object:Gem::Version
         version: '0.7'
-  type: :runtime
+  type: :development
   prerelease: false
   version_requirements: *70276294379100
 - !ruby/object:Gem::Dependency
